src/fragments/lib/logging/common/setup_logging/setup_logging.mdx

```diff
@@ -0,0 +1,95 @@
+The Amplify Logger enables you to troubleshoot and debug issues with your apps, to help you provide the best experience for your customers. You can log messages for errors by the Amplify library and add custom logs as well and send them to Amazon CloudWatch. With the Amplify Logger, you also can remotely change your logging configuration to adjust your logging levels, or add an allow list of customer IDs to help you detect issues more granularly for your apps in production.
+## Prerequisites
+
+import iosPreReq from '/src/fragments/lib/logging/ios/setup_logging/10_pre_req.mdx';
+import androidPreReq from '/src/fragments/lib/logging/android/setup_logging/10_pre_req.mdx';
+
+<Fragments
+  fragments={{
+    ios: iosPreReq,
+    android: androidPreReq
+  }}
+/>
+
+## Provision backend resources
+
+<Callout>
+
+You can use Amplify CLI generate a CDK template and [add custom resources](https://docs.amplify.aws/cli/custom/cdk/).  
+
+</Callout>
+
+You will need to create a log group in Amazon CloudWatch to send logs to.  You can create and provision a log group by going through the AWS Console and creating your log group manually or using Amplify CLI and AWS CDK to provision and deploy the AWS resources.
+
+Below is a sample CDK construct to create the Amazon CloudWatch log group as well as creating and assigning the permission policies to Amplify roles.
+
+The `<log-group-name>` and `<region` configured in the CDK construct will be used later to initialize the Amplify Logger plugin.
+
+```ts
+import * as cdk from "aws-cdk-lib"
+import { Construct } from "constructs"
+import * as logs from "aws-cdk-lib/aws-logs"
+import * as path from "path"
+import * as iam from "aws-cdk-lib/aws-iam"
+
+export class RemoteLoggingConstraintsConstruct extends Construct {
+  constructor(scope: Construct, id: string, props: RemoteLoggingConstraintProps) {
+    super(scope, id)
+
+    const region = cdk.Stack.of(this).region
+    const account = cdk.Stack.of(this).account
+    const logGroupName = <log-group-name>
+    
+    new logs.LogGroup(this, 'Log Group', {
+      logGroupName: logGroupName,
+      retention: logs.RetentionDays.INFINITE
+    })
+    
+    const authRole = iam.Role.fromRoleName(this, "Auth-Role", <amplify-authenticate-role-name>)
+    const unAuthRole = iam.Role.fromRoleName(this, "UnAuth-Role", <amplify-unauthenticated-role-name>)
+    const logResource = `arn:aws:logs:${region}:${account}:log-group:${logGroupName}:log-stream:*`
+    const logIAMPolicy = new iam.PolicyStatement({
+      effect: iam.Effect.ALLOW,
+      resources: [logResource],
+      actions: ["logs:PutLogEvents", "logs:DescribeLogStreams", "logs:CreateLogStream"]
+    })
+
+    authRole.addToPrincipalPolicy(logIAMPolicy)
+    unAuthRole.addToPrincipalPolicy(logIAMPolicy)
+
+    new cdk.CfnOutput(this, 'CloudWatchLogGroupName', { value: logGroupName });
+    new cdk.CfnOutput(this, 'CloudWatchRegion', { value: region });
+  }
+}
+```
+
```

Comment:
at the end of this - can we please add that the cloudwatchloggroupname and cloudwatchregion will be printed in the terminal, and that you can then use that information to set up the Amplify library later in this guide.

Reply:
Added text at the bottom after the code snippet.